components/BottomNavigation/src/private/MDCBottomNavigationLargeItemDialogView.m

```diff
@@ -0,0 +1,133 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCBottomNavigationLargeItemDialogView.h"
+
+#import <CoreGraphics/CoreGraphics.h>
+
+static const CGFloat kTitleFontScaling = (CGFloat)0.5;
+static const CGFloat kTitleFontSize = 35;
+static const CGFloat kImageViewHeightToDialogRatio = (CGFloat)0.35;
+
+/**
+ * Returns an image representing the given UITabBarItem. If the item has a largeContentSizeImage,
+ * this function will return that image, otherwise it returns the value of the item's image
+ * property. This value may be nil.
+ */
+static UIImage *_Nullable MDCImageForItem(UITabBarItem *_Nullable item) {
```

Comment:
Please make this API `nonnull` if possible.

Reply:
Done.  Made the `item` parameter `_Nonnull`, but the return value if nil by design.